user_profile/models.py

```diff
@@ -25,14 +27,16 @@ class UserProfile(models.Model):
 
     # Matching preference filters
     age_lower = models.IntegerField(
-        default=0,
+        default=18,
         verbose_name="Age(lower bound)",
         help_text="Enter lower bound for age filter",
+        validators=[MinValueValidator(18), MaxValueValidator(99)],
     )
     age_upper = models.IntegerField(
         default=99,
         verbose_name="Age(upper bound)",
         help_text="Enter upper bound for age filter",
+        validators=[MinValueValidator(18), MaxValueValidator(99)],
```

Comment:
Does min have a hard check on 18, is it only an integer check as opposed to a decimal?